app/serializers/api/v1/organisation_serializer.rb

```diff
@@ -15,5 +16,9 @@ def orders_count
     def orders_count__sql
       "(SELECT COUNT(*) FROM orders o WHERE o.organisation_id = orders.id"
     end
+
+    def include_orders_count?
+      !@options[:skip_orders]
```

Comment:
can we rename the option to `include_orders_count` ? I think `skip_orders` will lead people to believe that we're loading full orders

Reply:
Yes I actually thought that. If we want to include extra attributes/relationships, it would be better if this were generally always excluded by default and included via parameters rather than the other way around. opt-in vs opt-out

Reply:
This is done